queries/idl/highlights.scm

```diff
@@ -49,6 +49,28 @@
   "finder"
 ] @keyword.modifier
 
+[
+  "SEQUENTIAL"
+  "HASH"
+  "FINAL"
+  "APPENDABLE"
+  "MUTABLE"
+  "c"
+  "c++"
+  "java"
+  "idl"
+  "*"
```

Comment:
duplicated

Reply:
Can you be more detailed?

Reply:
There is another `"*"` on line 71, duplicated to this one.

Reply:
The `"*"` node appears at the end of the list again.